code/game/machinery/doors/airlock_types.dm

```diff
@@ -631,16 +631,27 @@
 	Misc Airlocks
 */
 
-/obj/machinery/door/airlock/glass_large
+/obj/machinery/door/airlock/multi_tile/glass
+	multi_tile = TRUE
 	name = "large glass airlock"
 	icon = 'icons/obj/doors/airlocks/glass_large/glass_large.dmi'
 	overlays_file = 'icons/obj/doors/airlocks/glass_large/overlays.dmi'
 	opacity = FALSE
-	assemblytype = null
+	assemblytype = /obj/structure/door_assembly/multi_tile/glass
 	glass = TRUE
 	bound_width = 64 // 2x1
 
-/obj/machinery/door/airlock/glass_large/narsie_act()
+/obj/structure/fluff/airlock_filler
```

Comment:
Wait why is this defined in a different file than its procs I hate this place
move the procs in here and add that documenting comment